### ISSUE TITLE
feat: multi language implementations/support

### ISSUE THREAD

**Issue by TyIsI** [vhs/raffle#12]
> Bring back the .js version, and any other languages, so results can be corroborated. Ditto there used to be a BASH version.

_Originally posted by @robmackenzie in https://github.com/vhs/raffle/issues/3#issuecomment-1622624283_

**Comment by TyIsI** [comment]
@robmackenzie you removed those lol

https://github.com/vhs/raffle/commit/642582ccd6285639496c28004e432c850e66d4f5

**Comment by TyIsI** [comment]
Also #2

**Comment by robmackenzie** [comment]
Yeah, I did remove the other versions as they were not in a working state. I would like to see them re-implemented one day, just so others can "prove" the algorithm easier, but it's low priority
